program/c/src/oracle/oracle.h

```diff
@@ -6,6 +6,13 @@
 extern "C" {
 #endif
 
+//a new custom return value to indicate to rust that aggregate was updated
+//this triggers SMA trackers to update
+//values 0-14 are defined in solana_sdk.h
+//used consts instead of define because bingen always turns
+// defines to u32 (even with ULL suffix)
+const uint64_t SUCCESSFULLY_UPDATED_AGGREGATE = 15ULL;
```

Comment:
The comment should be written in a timeless way, because you don't know when someone will read it. E.g., "new" isn't going to make sense a year in the future. I would stick with something like "A return value indicating that the aggregate price was updated" for the first sentence.

I suggest using a number >> 15 in order to ensure that the solana_sdk still has extensibility. what if they want to add an additional code?